scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -159,7 +159,7 @@
         (controller/update-expected-state
           this
           (:task-id task-metadata)
-          {:expected-state :expected/starting :launch-pod {:pod (api/task-metadata->pod task-metadata)
+          {:expected-state :expected/starting :launch-pod {:pod (api/task-metadata->pod pod-namespace task-metadata)
```

Comment:
There's a redundancy here. You're passing now pod-namespace within constructed pod (via the api/task-metadata->pod). However, launch-pod is extracting the namespace via the [:launch-pod :namespace] path.

Suggestion, strip the namespace out of the :launch-pod, and make launch-pod extract the namespace out of the V1Pod. This leaves :launch-pod dictionary with only one key, but I'm fine with that.

Reply:
Done